tests/python/unittest/test_meta_schedule_relay_integration.py

```diff
@@ -420,6 +420,7 @@ def test_extract_task_arm_conv2d_nchwc():
 
 
 def test_meta_schedule_te2primfunc_argument_order_and_lowering():
+    target = Target("llvm --num-cores=16")
```

Comment:
Just curious, what does that change do?

Reply:
Ach, this is a debugging artefact that I forgot to clean up, will remove. Good catch!